### PR TITLE
Feature002

### PR DESCRIPTION
**Pull request by wangyiqunumich**

### CODE REVIEW

File: src/main/java/com/laioffer/trademarket/HelloServlet.java [hunk 1]
```diff
@@ -19,6 +19,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         PrintWriter out = response.getWriter();
         out.println("<html><body>");
         out.println("<h1>" + message + "</h1>");
+        out.println("<h1>" + message + "</h1>");
```
Comment (JerryZhao1025) [inline src/main/java/com/laioffer/trademarket/HelloServlet.java:22]: You don't need this line!

File: src/main/java/com/laioffer/trademarket/HelloServlet.java [hunk 2]
```diff
@@ -9,7 +9,7 @@ public class HelloServlet extends HttpServlet {
     private String message;
 
     public void init() {
-        message = "Hello feature 001!";
+        message = "Hello feature 002!";
```
Comment (JerryZhao1025) [inline src/main/java/com/laioffer/trademarket/HelloServlet.java:12]: doing good

**Review by JerryZhao1025 — CHANGES_REQUESTED** [review]

File: src/main/java/com/laioffer/trademarket/HelloServlet.java (hunk 1 above)
Comment (wangyiqunumich) [reply src/main/java/com/laioffer/trademarket/HelloServlet.java:22]: Ok I agree

File: src/main/java/com/laioffer/trademarket/HelloServlet.java (hunk 2 above)
Comment (wangyiqunumich) [reply src/main/java/com/laioffer/trademarket/HelloServlet.java:12]: Thx

**Review by JerryZhao1025 — APPROVED** [review]

**Comment by JerryZhao1025** [comment]
+1

**Pull request merged**